src/internationalization/translations/en.json

```diff
@@ -293,6 +303,32 @@
     "primaryButton": "OK",
     "secondaryButton": ""
   },
+  "emergency": {
+    "header": "Emergency",
+    "emerButtonText": "SOS",
+    "callToAction": "Press to Report an Emergency Case",
+    "callToActionDesc": "After pressing the SOS button, we will contact all \nyour caretakersabout your emergency.",
+    "reportComplete": "Emergency Reported",
+    "reportCompleteDesc": "Your caretaker(s) have been notified.\n Please stand by.",
+    "callPolice": "Call police 191",
```

Comment:
Just curious. Does `\n` work in your screen? I remember trying it before but it doesn't work. If it returns a normal new line, then no problem.

Reply:
It actually works for me.